e2e/env.js

```diff
@@ -1,27 +0,0 @@
-module.exports = function () {
-
-  // see list of available hooks here
-  // https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/event_handlers.md
-
-  this.setDefaultTimeout(400 * 1000);
```

Comment:
That's because of the `.gitignore` file. I suggest opening up a PR to update that instead, and check the file back in. You can add it as an exception right after `/e2e/*.js`,  like `!/e2e/env.js`.